.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: php
+
+php:
+  - 5.5
+  - 5.6
+  - 7.0
+  - nightly
+
+matrix:
+  allow_failures:
+   - php: nightly
+  fast_finish: true
+
+cache:
+  directories:
+    - $HOME/.composer/cache
+
+install:
+  - composer install
+  - composer show --installed
```

Comment:
Yes, and Travis also does `self-update` now automatically. Removed `self-update`, but not that.